### PR TITLE
running WAR; removing workaround

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/employee-rostering/optashift-ER-runningwar-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+[source,bash]
+----
+EAP_HOME/bin/jboss-cli.sh --connect
+----
++
+.. In the management CLI enter the following command:
++
+[source]
+----
+/subsystem=ee:write-attribute(name="jboss-descriptor-property-replacement",value=VALUE)
```
Comment (rsynek) [inline doc-content/enterprise-only/employee-rostering/optashift-ER-runningwar-proc.adoc]: Should have been 'spec-descriptor-property-replacement'. The EAP documentation explains the difference between these two - each one of them targets different set of descriptor files.

File: doc-content/enterprise-only/employee-rostering/optashift-ER-building-eclipse-proc.adoc [hunk 2]
```diff
@@ -3,9 +3,10 @@
 You can use Eclipse, including Red Hat JBoss Development Studio, to build the employee rostering starter application and run it.
 
 .Prerequisites
-. Eclipse is installed.
-. The host has access to the Internet (for downloading Maven packages from external repositories).
-. Google Chrome is installed in order to run the application with the suggested configuration. However, you can modify the configuration to use another web browser.
+* You prepared the deployment files as described in <<er-deploy-prepare-proc>>.
+* Eclipse is installed.
+* The host has access to the Internet (for downloading Maven packages from external repositories).
```
Comment (emmurphy1) [inline doc-content/enterprise-only/employee-rostering/optashift-ER-building-eclipse-proc.adoc]: Just an observation. The IBM SG says not to use parenthesis unless abs necessary. I would remove these and consider this all of your docs.

File: doc-content/enterprise-only/employee-rostering/optashift-ER-building-eclipse-proc.adoc [hunk 3]
```diff
@@ -3,9 +3,10 @@
...
+* Google Chrome is installed in order to run the application with the suggested configuration. However, you can modify the configuration to use another web browser.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/employee-rostering/optashift-ER-building-eclipse-proc.adoc]: I would make the second sentence a note. Note: You can modify...

File: doc-content/enterprise-only/employee-rostering/optashift-ER-runningwar-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,30 @@
+[id='optashift-ER-runningwar-proc']
+= Running the supplied pre-built WAR file
+
+You can deploy and run the pre-built `employee-rostering-webapp-{MAVEN_ARTIFACT_VERSION}.war` file that is supplied with {PRODUCT}.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/employee-rostering/optashift-ER-runningwar-proc.adoc]: Can you expand this intro to say why? What does this file do?

File: doc-content/enterprise-only/employee-rostering/optashift-ER-runningwar-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+.. In the management CLI enter the following command:
++
+[source]
+----
+/subsystem=ee:write-attribute(name="spec-descriptor-property-replacement",value=VALUE)
+----
++
+. Stop the application server.
+. Deploy the `binaries/employee-rostering-webapp-{MAVEN_ARTIFACT_VERSION}.war` directory into the application server and start the application server.
+. Access the application at http://localhost:8080/gwtui/gwtui.html
```
Comment (emmurphy1) [inline doc-content/enterprise-only/employee-rostering/optashift-ER-runningwar-proc.adoc]: Needs a full stop. But I would reword something like:
To start the application, enter BLAH in a web browser.
